### ISSUE TITLE
Error when running examples

### ISSUE THREAD

**Issue by Yam76** [gtk-rs/examples#278]
Received this error when running some examples. Curiously enough, this did not happen originally, as the first few examples I tried did not produce any errors.
```
(accessibility:3616): GLib-GIO-CRITICAL **: 12:29:00.830: g_dbus_proxy_new_sync: assertion 'G_IS_DBUS_CONNECTION (connection)' failed
```

**Comment by Yam76** [comment]
The answer seems to be found [here](https://askubuntu.com/a/1104648). Basically, run `ps aux` or `ps -A`. Chances are that you'll see that a dbus process is already running. Use `sudo kill PID` to kill the process and the errors should stop.
